src/main/scala/org/mdedetrich/apache/sonatype/SonatypeApachePlugin.scala

```diff
@@ -66,10 +66,11 @@ object SonatypeApachePlugin extends AutoPlugin {
         )
       }
     },
-    organizationName             := apacheSonatypeOrganizationName.value,
-    organizationHomepage         := Some(apacheSonatypeOrganizationHomePage.value),
-    publishMavenStyle            := true,
```

Comment:
These are removed since they are set by sbt-sonatype however I have added a test to make sure they are properly set. Maybe there is still a reason to explicitly set them again since in sbt-sonatype they are set as `projectSettings` where as for this plugin we ideally should set it as `buildSettings` since we wont to globally enforce it in the build?